central/networkgraph/entity/datastore/datastore_impl_test.go

```diff
@@ -591,7 +590,7 @@ func (suite *NetworkEntityDataStoreTestSuite) TestDefaultGraphSetting() {
 		} else {
 			pushSig = suite.expectPushExternalNetworkEntitiesToSensor(cluster)
 		}
-		suite.NoError(suite.ds.DeleteExternalNetworkEntity(suite.globalWriteAccessCtx, entity.GetInfo().GetId()))
+		suite.NoError(suite.ds.DeleteExternalNetworkEntity(suite.elevatedCtx, entity.GetInfo().GetId()))
```

Comment:
Is this context change still needed ? By the look of things, both contexts are identical : global R+RW on NetworkGraph